FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,14 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		
+		client.get(from: url) { [weak self] (result) in
+			if self == nil { return }
+			switch result {
+			case .success((let data, let response)):
```

Comment:
You can destructure the values with a single `let` instead of two:

```suggestion
			case let .success((data, response)):
```